solutions/client-landing-zone/client-folder/folder-sink.yaml

```diff
@@ -37,6 +37,7 @@ spec:
       # Only `external` field is supported to configure the reference.
       external: platform-and-component-client-name-log-bucket # kpt-set: logging.googleapis.com/projects/${logging-project-id}/locations/northamerica-northeast1/buckets/platform-and-component-${client-name}-log-bucket
   description: Folder sink for client-name Platform and Component logs # kpt-set: Folder sink for ${client-name} Platform and Component logs
+  disabled: false
```

Comment:
Think that's a good idea. Having some context on why it's set to false would be a great add.

Reply:
I was thinking something similar because of the "double negative", but opted not to because it's not exposed via setters and noticed `disabled: false` also exists for each exclusion filters...

but can't hurt, so does this make sense?
```yaml
# the log sink must be enabled (disabled: false) to meet the listed security controls
```

Reply:
I agree with the comment you chose.

Reply:
I think that ought to work.

Reply:
comment added to all sinks as discussed